### PR TITLE
add 6.1.0 release notes

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran.alex@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add 6.1.0 release notes

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-6.1.0.md [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+- TiCDC 支持将 changelogs 以 Avro 协议输出到 Kafka
+- DM 合库合表迁移场景的乐观 DDL 协调模式正式 GA
+
+* 注： TiFlash 6.1 引入了较多新功能，为方便顺畅升级，请参考 [TiFlash v6.1.0 升级帮助](/tiflash-610-upgrade.md)。*
+
+## 新功能
+
+### SQL
+
+* List, List columns 分区方式正式 GA，与MySQL 5.7 兼容。
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:29]:
```suggestion
* List, List columns 分区方式正式 GA，与 MySQL 5.7 兼容。
```

File: releases/release-6.1.0.md [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+
+    [用户文档：List 分区](/partitioned-table.md#list-分区)，[用户文档：List COLUMNS 分区](/partitioned-table.md#list-columns-分区)
+
+* 支持通过 SQL 语句对 TiFlash 副本立即触发物理数据整理（Compaction）。
+
+    在当前 TiFlash 后台自动整理物理数据（Compaction）机制基础上，新增 compact  命令,帮助刷新旧格式数据，提升读写性能。推荐在升级至 v6.1.0 之后，执行该语句以清理数据。此语句是对标准 SQL 语法的扩展，对 MySQL 客户端保持兼容。升级之外场景一般不需要特别关注。
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:35]:
```suggestion
    在当前 TiFlash 后台自动整理物理数据（Compaction）机制基础上，新增 compact 命令,帮助刷新旧格式数据，提升读写性能。推荐在升级至 v6.1.0 之后，执行该语句以清理数据。此语句是对标准 SQL 语法的扩展，对 MySQL 客户端保持兼容。升级之外场景一般不需要特别关注。
```

File: releases/release-6.1.0.md [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+
+* 发布 Raft Engine 存储引擎
+
+    TiDB 从 v6.1.0 开始默认使用引擎 [Raft Engine](https://github.com/tikv/raft-engine) 作为 TiKV 的日志存储引擎。新引擎与 RocksDB 相比，可以减少至多 40% 的 TiKV I/O 写流量和 10% 的 CPU 使用，同时在特定负载下提升 5% 左右前台吞吐，减少 20% 长尾延迟。
+
+    [用户文档](/tikv-configuration-file.md#raft-engine)，[#95](https://github.com/tikv/raft-engine/issues/95)
+
+* 支持 join order hint 语法
+
+    * LEADING hint: 提示优化器使用指定的顺序作为连接前缀，好的连接前缀顺序可以在连接初期快速地降低数据量，提升查询性能。
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:75]:
```suggestion
    * LEADING hint：提示优化器使用指定的顺序作为连接前缀，好的连接前缀顺序可以在连接初期快速地降低数据量，提升查询性能。
```

File: releases/release-6.1.0.md [hunk 4]
```diff
@@ -0,0 +1,441 @@
...
+
+        [用户文档](/optimizer-hints.md#leadingt1_name--tl_name-)
+
+    * STRAIGHT_JOIN hint: 提示优化器按照表在 `FROM` 子句中的出现顺序进行连接。
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:79]:
```suggestion
    * STRAIGHT_JOIN hint：提示优化器按照表在 `FROM` 子句中的出现顺序进行连接。
```

File: releases/release-6.1.0.md [hunk 5]
```diff
@@ -0,0 +1,441 @@
...
+
+    通过本次优化，客户可以在线修改参数生效并持久化，方便客户进行日常运维，避免重启生效对在线业务造成的影响。
+
+    [用户文档](/dynamic-config.md)
+
+* 支持全局 kill 查询或连接
+
+    支持通过 `enable-global-kill` 配置项（默认开启）设置全局 kill 开关。
+
+    在 TiDB v6.1.0 之前，当某个特定操作占用大量资源引发集群稳定性问题时，你需要先登陆到对应的 TiDB 节点，然后运行 `kill [TiDB] id` 命令终止对应的连接及操作。在 TiDB 节点多的情况下，这种方式使用不便，并且容易误操作。从 v6.1.0 起，当开启`enable-global-kill`  配置项时，你可以在任意 TiDB 节点运行 kill 命令终止指定的连接及操作，而无需担心客户端和 TiDB 中间有代理时错误地终止其他查询或会话。目前 TiDB 暂时不支持用 Ctrl+C 终止查询或会话。
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:171]:
```suggestion
    在 TiDB v6.1.0 之前，当某个特定操作占用大量资源引发集群稳定性问题时，你需要先登陆到对应的 TiDB 节点，然后运行 `kill [TiDB] id` 命令终止对应的连接及操作。在 TiDB 节点多的情况下，这种方式使用不便，并且容易误操作。从 v6.1.0 起，当开启 `enable-global-kill`  配置项时，你可以在任意 TiDB 节点运行 kill 命令终止指定的连接及操作，而无需担心客户端和 TiDB 中间有代理时错误地终止其他查询或会话。目前 TiDB 暂时不支持用 Ctrl+C 终止查询或会话。
```

File: releases/release-6.1.0.md [hunk 6]
```diff
@@ -0,0 +1,441 @@
...
+
+    [用户文档](/tidb-configuration-file.md#enable-global-kill-从-v610-版本开始引入)，[#8854](https://github.com/pingcap/tidb/issues/8854)
+
+* TiKV API v2
+
+    在 v6.1.0 之前，TiKV 作为 Raw Key Value 存储时，由于仅存储了客户端传入的原始数据，因此只提供基本的 Key Value 读写能力。
+
+    TiKV API v2 提供了新的 Raw Key Value 存储格式与访问接口，包括：
+
+    * 数据以 MVCC（Multi Version Concurrency Control）方式存储，并记录了数据的变更时间戳。这个特性将为实现 Change Data Capture、增量备份与恢复等打下基础。
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:181]:
```suggestion
    * 数据以 MVCC（Multi-Version Concurrency Control）方式存储，并记录了数据的变更时间戳。这个特性将为实现 Change Data Capture、增量备份与恢复等打下基础。
```

File: releases/release-6.1.0.md [hunk 7]
```diff
@@ -0,0 +1,441 @@
...
+    * 数据根据不同的使用方式划分范围，支持单一集群 TiDB / 事务 KV / RawKV 应用共存。
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:182]:
```suggestion
    * 数据根据不同的使用方式划分范围，支持单一集群 TiDB、事务 KV、RawKV 应用共存。
```

File: releases/release-6.1.0.md [hunk 8]
```diff
@@ -0,0 +1,441 @@
...
+
+* 支持全局 kill 查询或连接
+
+    支持通过 `enable-global-kill` 配置项（默认开启）设置全局 kill 开关。
+
+    在 TiDB v6.1.0 之前，当某个特定操作占用大量资源引发集群稳定性问题时，你需要先登陆到对应的 TiDB 节点，然后运行 `kill [TiDB] id` 命令终止对应的连接及操作。在 TiDB 节点多的情况下，这种方式使用不便，并且容易误操作。从 v6.1.0 起，当开启`enable-global-kill`  配置项时，你可以在任意 TiDB 节点运行 kill 命令终止指定的连接及操作，而无需担心客户端和 TiDB 中间有代理时错误地终止其他查询或会话。目前 TiDB 暂时不支持用 Ctrl+C 终止查询或会话。
+
+    [用户文档](/tidb-configuration-file.md#enable-global-kill-从-v610-版本开始引入)，[#8854](https://github.com/pingcap/tidb/issues/8854)
+
+* TiKV API v2
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:175]:
```suggestion
* TiKV API V2
```

File: releases/release-6.1.0.md [hunk 9]
```diff
@@ -0,0 +1,441 @@
...
+
+    在 v6.1.0 之前，TiKV 作为 Raw Key Value 存储时，由于仅存储了客户端传入的原始数据，因此只提供基本的 Key Value 读写能力。
+
+    TiKV API v2 提供了新的 Raw Key Value 存储格式与访问接口，包括：
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:179]:
```suggestion
    TiKV API V2 提供了新的 Raw Key Value 存储格式与访问接口，包括：
```

File: releases/release-6.1.0.md [hunk 10]
```diff
@@ -0,0 +1,441 @@
...
+
+    注意：由于底层存储格式发生了重大变化，启用 API V2 后，不能将 TiKV 集群回退到 v6.1.0 之前的版本，否则可能导致数据损坏。
+
+    [用户文档](/tikv-configuration-file.md#api-version-从-v610-版本开始引入)，[#11745](https://github.com/tikv/tikv/issues/11745)
+
+### MySQL 兼容性
+
+* 支持兼容 MySQL 的用户级别锁管理
+
+    用户级别锁是 MySQL 通过内置函数提供的用户命名锁管理系统。它们可以提供锁阻塞，等待，等锁管理能力。用户级别锁在 ORM 框架中也有较为广泛的应用，例如 RoR, Elixir 和 Ecto 等。TiDB 从 v6.1.0 版本开始支持兼容 MySQL 的用户级别锁管理，支持 `GET_LOCK`，`RELEASE_LOCK`, `RELEASE_ALL_LOCKS` 函数。
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:192]:
```suggestion
    用户级别锁是 MySQL 通过内置函数提供的用户命名锁管理系统。它们可以提供锁阻塞，等待，等锁管理能力。用户级别锁在 ORM 框架中也有较为广泛的应用，例如 RoR，Elixir 和 Ecto 等。TiDB 从 v6.1.0 版本开始支持兼容 MySQL 的用户级别锁管理，支持 `GET_LOCK`，`RELEASE_LOCK`，`RELEASE_ALL_LOCKS` 函数。
```

File: releases/release-6.1.0.md [hunk 11]
```diff
@@ -0,0 +1,441 @@
...
+
+    + TiDB Lightning
+
+        - 优化 Scatter Region 为批量模式，提升 Scatter Region 过程的稳定性 [#33618](https://github.com/pingcap/tidb/issues/33618)
+
+## 错误修复
+
++ TiDB
+
+    - 修复 `in` 函数处理 `bit` 数据类型时可能会导致 TiDB panic 的问题 [#33070](https://github.com/pingcap/tidb/issues/33070)
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:377]:
```suggestion
    - 修复 `IN` 函数处理 `BIT` 数据类型时可能会导致 TiDB panic 的问题 [#33070](https://github.com/pingcap/tidb/issues/33070)
```

File: releases/release-6.1.0.md [hunk 12]
```diff
@@ -0,0 +1,441 @@
...
+| [`tidb_prepared_plan_cache_size`](/system-variables.md#tidb_prepared_plan_cache_size-从-v610-版本开始引入) | 新增 | 由 TiDB 配置项 `prepared-plan-cache.capacity` 转化而来。 |
+| [`tidb_stats_cache_mem_quota`](/system-variables.md#tidb_stats_cache_mem_quota-从-v610-版本开始引入) | 新增 | 控制 TiDB 内部统计信息缓存使用内存的上限。 |
+
+### 配置文件参数
+
+| 配置文件 | 配置项 | 修改类型 | 描述 |
+|:---|:---|:---|:-----|
+| TiDB | `committer-concurrency` | 删除 | 转化为系统变量 `tidb_committer_concurrency` |
+| TiDB | `enable-batch-dml` | 删除 | 转化为系统变量 `tidb_enable_batch_dml` |
+| TiDB | `lower-case-table-names` | 删除 | TiDB 目前只支持`lower_case_table_name=2`, 如果升级前设置了其他值，升级到 v6.1.0 后该值会丢失。 |
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:262]:
```suggestion
| TiDB | `lower-case-table-names` | 删除 | TiDB 目前只支持`lower_case_table_name=2`，如果升级前设置了其他值，升级到 v6.1.0 后该值会丢失。 |
```

File: releases/release-6.1.0.md [hunk 13]
```diff
@@ -0,0 +1,441 @@
...
+| TiDB | `prepared-plan-cache.enabled` | 删除 | 转化为系统变量 `tidb_enable_prepared_plan_cache` |
+| TiDB | `query-log-max-len` | 删除 | 转化为系统变量 `tidb_query_log_max_len` |
+| TiDB | `require-secure-transport` | 删除 | 转化为系统变量 `require_secure_transport` |
+| TiDB | `run-auto-analyze` | 删除 | 转化为系统变量 `tidb_enable_auto_analyze` |
+| TiDB | [`enable-global-kill`](/tidb-configuration-file.md#enable-global-kill-从-v610-版本开始引入) | 新增 | 当该配置项值默认为 `true` ，`KILL` 语句和 `KILL TIDB` 语句均能跨节点终止查询或连接，无需担心错误地终止其他查询或连接。 |
+| TiDB | [`enable-stats-cache-mem-quota`](/tidb-configuration-file.md#enable-stats-cache-mem-quota-从-v610-版本开始引入) | 新增 | 控制 TiDB 是否开启统计信息缓存的内存上限。 |
+| TiKV | [`raft-engine.enable`](/tikv-configuration-file.md#enable-1) | 修改 | 默认值从 `false` 修改为 `true`。 |
+| TiKV | [`region-max-keys`](/tikv-configuration-file.md#region-max-keys) | 修改 | 默认值从 1440000 修改为 `region-split-keys / 2 * 3`。 |
+| TiKV | [`region-max-size`](/tikv-configuration-file.md#region-max-size) | 修改 | 默认值从 144 MB 修改为 `region-split-size / 2 * 3`。 |
+| TiKV | [`coprocessor.enable-region-bucket`](/tikv-configuration-file.md#enable-region-bucket-从-v610-版本开始引入) | 新增 | 控制是否开启 region bucket 将 region 划分为更小的区间。 |
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:275]:
```suggestion
| TiKV | [`coprocessor.enable-region-bucket`](/tikv-configuration-file.md#enable-region-bucket-从-v610-版本开始引入) | 新增 | 是否将 Region 划分为更小的区间 bucket。 |
```

File: releases/release-6.1.0.md [hunk 14]
```diff
@@ -0,0 +1,427 @@
...
+
+## 新功能
+
+### SQL
+
+* List, List columns 分区方式正式 GA，与 MySQL 5.7 兼容。
+
+    [用户文档：List 分区](/partitioned-table.md#list-分区)，[用户文档：List COLUMNS 分区](/partitioned-table.md#list-columns-分区)
+
+* 支持通过 SQL 语句对 TiFlash 副本立即触发物理数据整理（Compaction）。
```
Comment (ran-huang) [inline releases/release-6.1.0.md:30]:
```suggestion
* 支持通过 SQL 语句对 TiFlash 副本立即触发物理数据整理 (Compaction)（实验特性）
```

File: releases/release-6.1.0.md [hunk 15]
```diff
@@ -0,0 +1,427 @@
...
+| TiKV | [`coprocessor.enable-region-bucket`](/tikv-configuration-file.md#enable-region-bucket-从-v610-版本开始引入) | 新增 | 是否将 Region 划分为更小的区间 bucket。 |
+| TiKV | [`coprocessor.region-bucket-size`](/tikv-configuration-file.md#region-bucket-size-从-v610-版本开始引入) | 新增 | 设置 `enable-region-bucket` 启用时 bucket 的预期大小。 |
+| TiKV | [`causal-ts.renew-batch-min-size`](/tikv-configuration-file.md#renew-batch-min-size) | 新增 | 时间戳缓存的最小数量。 |
+| TiKV | [`causal-ts.renew-interval`](/tikv-configuration-file.md#renew-interval) | 新增 | 刷新本地缓存时间戳的周期。 |
+| TiKV | [`max-snapshot-file-raw-size`](/tikv-configuration-file.md#max-snapshot-file-raw-size-从-v610-版本开始引入) | 新增 | 当 snapshot 文件大于该配置项指定的大小时，snapshot 文件会被切割为多个文件。 |
+| TiKV | [`raft-engine.memory-limit`](/tikv-configuration-file.md#memory-limit) | 新增 | 指定 Raft Engine 使用内存的上限。 |
+| TiKV | [`storage.background-error-recovery-window`](/tikv-configuration-file.md#background-error-recovery-window-从-v610-版本开始引入) | 新增 | RocksDB 检测到可恢复的后台错误后，所允许的最长恢复时间。 |
+| TiKV | [`storage.block-cache.api-version`](/tikv-configuration-file.md#api-version-从-v610-版本开始引入) | 新增 | TiKV 作为 Raw Key Value 存储数据时使用的存储格式与接口版本。 |
```
Comment (pingyu) [inline releases/release-6.1.0.md:271]:
```suggestion
| TiKV | [`storage.api-version`](/tikv-configuration-file.md#api-version-从-v610-版本开始引入) | 新增 | TiKV 作为 Raw Key Value 存储数据时使用的存储格式与接口版本。 |
```

是 `tikv-configuration-file.md` 错了，我提个 PR 改过来。

File: releases/release-6.1.0.md [hunk 16]
```diff
@@ -0,0 +1,427 @@
...
+    [用户文档](/character-set-gbk.md#组件兼容性)，[#4806](https://github.com/pingcap/tiflow/issues/4806)
+
+## 兼容性变更
+
+### 系统变量
+
+| 变量名 | 修改类型 | 描述 |
+|:---|:--|:----|
+| [`tidb_enable_list_partition`](/system-variables.md#tidb_enable_list_partition-从-v50-版本开始引入) | 修改 | 默认值从 `OFF` 改为 `ON`。 |
+| [`tidb_mem_quota_query`](/system-variables.md#tidb_mem_quota_query) | 修改 | 增加 Global 作用域，变量值可以持久化到集群。 |
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:226]:
```suggestion
| [`tidb_mem_quota_query`](/system-variables.md#tidb_mem_quota_query) | 修改 | 增加 GLOBAL 作用域，变量值可以持久化到集群。 |
```

File: releases/release-6.1.0.md [hunk 17]
```diff
@@ -0,0 +1,427 @@
...
+| [`tidb_prepared_plan_cache_size`](/system-variables.md#tidb_prepared_plan_cache_size-从-v610-版本开始引入) | 新增 | 由 TiDB 配置项 `prepared-plan-cache.capacity` 转化而来。 |
+| [`tidb_stats_cache_mem_quota`](/system-variables.md#tidb_stats_cache_mem_quota-从-v610-版本开始引入) | 新增 | 控制 TiDB 内部统计信息缓存使用内存的上限。 |
+
+### 配置文件参数
+
+| 配置文件 | 配置项 | 修改类型 | 描述 |
+|:---|:---|:---|:-----|
+| TiDB | `committer-concurrency` | 删除 | 转化为系统变量 `tidb_committer_concurrency`。该配置项不再生效，如需修改，需修改对应的系统变量。 |
+| TiDB | `enable-batch-dml` | 删除 | 转化为系统变量 `tidb_enable_batch_dml`。该配置项不再生效，如需修改，需修改对应的系统变量。 |
+| TiDB | `lower-case-table-names` | 删除 | TiDB 目前只支持`lower_case_table_name=2`，如果升级前设置了其他值，升级到 v6.1.0 后该值会丢失。 |
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:251]:
```suggestion
| TiDB | `lower-case-table-names` | 删除 | TiDB 目前只支持 `lower_case_table_name=2`，如果升级前设置了其他值，升级到 v6.1.0 后该值会丢失。 |
```

File: releases/release-6.1.0.md (hunk 15 above)
Comment (pingyu) [reply releases/release-6.1.0.md:271]: https://github.com/pingcap/docs-cn/pull/9718

**Review by TomShawn — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**